dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.mpMetrics-monitor-2.0.feature

```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.mpMetrics-monitor-2.0
```

Comment:
This name looks weird to me, maybe `com.ibm.websphere.appserver.mpMetrics2.0-monitor1.0` would be better?

Reply:
Ah thanks, that would be a better idea